securedrop_client/gui/widgets.py

```diff
@@ -3704,13 +4196,13 @@ def __init__(self, source, controller):
         header_layout.addWidget(menu, alignment=Qt.AlignRight)
 
         # Create horizontal line
-        horizontal_line = QWidget()
-        horizontal_line.setObjectName("SourceProfileShortWidget_horizontal_line")
-        horizontal_line.setSizePolicy(QSizePolicy.Expanding, QSizePolicy.Fixed)
+        self.horizontal_line = QWidget()
+        self.horizontal_line.setObjectName("SourceProfileShortWidget_horizontal_line")
+        self.horizontal_line.setSizePolicy(QSizePolicy.Expanding, QSizePolicy.Fixed)
 
         # Add widgets
         layout.addWidget(header)
-        layout.addWidget(horizontal_line)
+        layout.addWidget(self.horizontal_line)
```

Comment:
why make this change?

Reply:
I'll revert it. I believe I was trying to manipulate it from elsewhere, but figured out how to do what I wanted with CSS instead.